examples/buttons/index.html

```diff
@@ -90,23 +96,25 @@ <h2 id="button with icon"><a href="#button-with-icon">button with icon</a></h2>
       )
     })
   </script>
-  <pre id="viewSource-button-with-icon"></pre><script>showSource('button-with-icon')</script>
+  <pre id="viewSource-button-with-icon"></pre>
+  <script>showSource('button-with-icon')</script>
   <div id="div-button-with-icon"></div>
 
- <h2 id="button-with-text"><a href="#button-with-text">button with text</a></h2>
+  <h2 id="button-with-text"><a href="#button-with-text">button with text</a></h2>
   <script id="script-button-with-text">
     window.addEventListener('DOMContentLoaded', (event) => {
       $('#div-button-with-text').appendChild(
         UI.widgets.button(document, undefined, 'test', () => {
           window.alert('clicked!')
-        })
+        }, { buttonType: UI.widgets.ButtonType.Secondary, filled: true })
```

Comment:
Please split this into 4 examples:
* primary, filled
* primary, not filled
* secondary, filled
* secondary, not filled